### PR TITLE
fix(chat): enabled "everyone" in group chat mentions

### LINKED ISSUE #11029: Chat input suggestion popup in ad-hoc group chats doesn't suggest "@everyone"
Something I've noticed in the latest 0.12.0 release.
`@everyone` is no longer suggested.

Screenshot from 2023-06-12 13-10-08

There's this issue: https://github.com/status-im/status-desktop/issues/11005 which touches on similar things, however it reports the exact opposite behaviour:

> The only thing that actually gets suggested is @everyone

@caybro feel free to close this if you're certain this was fixed as part of #11005 but judging by its description it does not.

### PR DESCRIPTION
**Pull request by alexandraB99**
Closes #11029

### What does the PR do
Enables "everyone" suggestion in group chat mentions box

### Affected areas
group chat mentions box

### Screenshot of functionality (including design for comparison)

https://github.com/status-im/status-desktop/assets/31625338/03903100-ac8e-4c92-95df-1c9de8c05e11

### CODE REVIEW

File: ui/app/AppLayouts/Chat/panels/SuggestionFilterPanel.qml [hunk 1]
```diff
@@ -86,7 +85,7 @@ Item {
             name: "everyone",
```
Comment (alexjba) [inline ui/app/AppLayouts/Chat/panels/SuggestionFilterPanel.qml:85]: I guess it shouldn't be translated, right? 🤔

**Review by alexjba — APPROVED** [review]
LGTM!

File: ui/app/AppLayouts/Chat/panels/SuggestionFilterPanel.qml (hunk 1 above)
Comment (alexandraB99) [reply ui/app/AppLayouts/Chat/panels/SuggestionFilterPanel.qml:85]: :|

**Review by micieslak — APPROVED** [review]
LGTM

File: ui/app/AppLayouts/Chat/panels/SuggestionFilterPanel.qml (hunk 1 above)
Comment (caybro) [reply ui/app/AppLayouts/Chat/panels/SuggestionFilterPanel.qml:85]: nah, it shouldn't imho

**Review by caybro — APPROVED** [review]
LGTM

**Pull request merged**